cms/templates/partials/enterprise/learning_journey.html

```diff
@@ -0,0 +1,24 @@
+{% load static image_version_url wagtailcore_tags %}
+
+<div class="learning-journey-block">
+    <div class="container">
+        <h1>{{ page.heading|richtext }}</h1>
```

Comment:
Do we need the h1 tag around the rich text?

Reply:
If we need these tags for styles, we can use `div` with classes. I can see multiple usages of rich text with `p` or `h1`